python/practices/practice-3/src/path1/word_e_checker.py

```diff
@@ -0,0 +1,53 @@
+"""
+Count and print percentage word in word list not contains the letter 'e' or 'E'
+"""
+
+
+import os
+
+
+def count_words(words):
+    """
+    Count and calculate the percentage of words that do not contain 'E' or 'e'
+    in a list of words.
+
+    Parameters:
+        - words: list of words input
+    """
+
+    words_has_no_e = 0
+
+    for word in words:
+        if 'e' not in word and 'E' not in word:
+            print(word)
+            words_has_no_e += 1
+
+    return words_has_no_e
+
+
+def percentage_words(words):
+    """
+    Calculate the percentage of words that do not contain 'E' or 'e'
+    in a list of words
+
+    Parameters:
+        - words: list of words input
+        - words_has_no_e: list of words not contain letter 'e' or 'E'
+    """
+
+    total_number_of_words = len(words)
+    words_has_no_e = count_words(words)
+    percentage = (words_has_no_e/total_number_of_words) * 100
```

Comment:
Have you thought about the case len(words) = 0?

Reply:
Done